README.md

```diff
@@ -36,6 +36,19 @@ reporting issues, contribute to the documentation and create sample gallery proj
   - [🦄️ Yatai](https://github.com/bentoml/yatai): Model Deployment at scale on Kubernetes
   - [🚀 bentoctl](https://github.com/bentoml/bentoctl): Fast model deployment on any cloud platform
 
+### Creating Bento with custom BentoML source
+
+It is possible to force a Bento to use a custom BentoML source distribution:
+
+1. Install custom BentoML in editable mode. e.g.:
+   * git clone your bentoml fork
+   * `pip install -e PATH_TO_THE_FORK`
+2. Set env var `export BENTOML_BUNDLE_LOCAL_BUILD=True`
+   * make sure you have the latest setuptools installed:  `pip install -U setuptools`
+3. Build a new Bento with `bentoml build` in your project directory
+4. The new Bento will include a wheel file built from the BentoML source, and 
+`bentoml containrize` will install it to override the default BentoML installation in base image
+
```

Comment:
I don't think this section should live under the main README.md. this should stay in the DEVELOPMENT.md

Reply:
The reasoning for this is that it's not really developers who want to do this, it's probably an end user who wants to try out changes from a PR. We might want to group it with the rest of the installation instructions, though.

Reply:
will move this back to development.md for now